samcli/lib/build/workflow_config.py

```diff
@@ -140,6 +140,8 @@ def get_layer_subfolder(runtime):
         "ruby2.7": "ruby/lib",
         "java8": "java",
         "java11": "java",
+        # User is responsible for creating subfolder in these workflows
```

Comment:
Nit: we may need to change the name of this method later, since makefile is not a runtime.

it could be `get_layer_subfolder(build_workflow)` maybe?

Reply:
Sure.